### ISSUE TITLE
LoadError in 0.4

### ISSUE THREAD

**Issue by dpo** [timholy/FastAnonymous.jl#7]
````
julia> VERSION
v"0.4.0-dev+4559"

julia> using FastAnonymous

WARNING: deprecated syntax "stagedfunction" at /Users/dpo/.julia/v0.4/FastAnonymous/src/FastAnonymous.jl:27.
Use "@generated function" instead.

WARNING: deprecated syntax "stagedfunction" at /Users/dpo/.julia/v0.4/FastAnonymous/src/FastAnonymous.jl:76.
Use "@generated function" instead.

WARNING: deprecated syntax "stagedfunction" at /Users/dpo/.julia/v0.4/FastAnonymous/src/FastAnonymous.jl:110.
Use "@generated function" instead.
ERROR: LoadError: MethodError: `start` has no method matching start(::Type{Symbol})
 in append_any at base.jl:198
 in include at /Applications/Julia-0.4.0-dev-060e5090e7.app/Contents/Resources/julia/lib/julia/sys.dylib
 in include_from_node1 at /Applications/Julia-0.4.0-dev-060e5090e7.app/Contents/Resources/julia/lib/julia/sys.dylib
 in reload_path at /Applications/Julia-0.4.0-dev-060e5090e7.app/Contents/Resources/julia/lib/julia/sys.dylib
 in _require at /Applications/Julia-0.4.0-dev-060e5090e7.app/Contents/Resources/julia/lib/julia/sys.dylib
 in require at /Applications/Julia-0.4.0-dev-060e5090e7.app/Contents/Resources/julia/lib/julia/sys.dylib
while loading /Users/dpo/.julia/v0.4/FastAnonymous/src/FastAnonymous.jl, in expression starting on line 25
````

**Comment by timholy** [comment]
Yep, just reported on julia-users, too: https://groups.google.com/d/msg/julia-users/PxNerI7kq9U/tBaOan-zMx0J. This one's not quite as easy as most other post-tupocalypse fixes, unfortunately, so it will take a me a few days to get around to looking at it in detail.

**Comment by timholy** [comment]
Fixed thanks to @malmaud in #9.
